user/xfsdump/template.py

```diff
@@ -0,0 +1,27 @@
+pkgname = "xfsdump"
+pkgver = "3.2.0"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_args = ["--enable-lib64=no"]
+# autoheader breaks definitions in include/config.h.in
+configure_gen = []
+# build system expects to be in the root of the source directory
+make_dir = "."
+make_install_args = ["PKG_ROOT_SBIN_DIR=/usr/bin"]
+hostmakedepends = ["gettext", "libuuid-devel"]
+makedepends = ["ncurses-devel", "xfsprogs-devel", "linux-headers", "attr-devel"]
+pkgdesc = "XFS Dump Tools"
+maintainer = "nilfsuser5678 <chimera-dev.o6dyz@passmail.net>"
+license = "GPL-2.0-only"
+url = "https://git.kernel.org/pub/scm/fs/xfs/xfsdump-dev.git"
+source = f"$(KERNEL_SITE)/utils/fs/xfs/xfsdump/xfsdump-{pkgver}.tar.xz"
+sha256 = "2914dbbe1ebc88c7d93ad88e220aa57dabc43d216e11f06221c01edf3cc10732"
+# there are asserts that dont compile when DEBUG isn't defined so they need to be disabled
+tool_flags = {"CFLAGS": ["-DNDEBUG", "-D_LARGEFILE64_SOURCE"]}
+hardening = ["vis", "cfi-icall", "int"]
```

Comment:
remove this, it doesn't do anything

Reply:
Sorry, I misunderstood the packaging.md bit about this. Thanks.

Reply:
cfi-icall is only a modifier and is by default on, so it won't do anything unless cfi is on also

int is on by default

vis is useless on its own